### DISCUSSION TITLE
MOVED redirection loop detected

### DISCUSSION THREAD

**Discussion by gufeng0** [redisson/redisson#5306] [Q&A] [answered]
I found this error when deploying the application. When I change the spring boot configuration from redis stand-alone to redis cluster, no error is reported. Why is this happening?

**Comment by gufeng0** [comment]
When I use this :
spring:
redis:
host: redis-node1-host
port: 6379
password: your-redis-password
It will report an error,
After I modified:
spring:
  redis:
    cluster:
      nodes: redis-node1-host:6379,redis-node2-host:6379
    password: your-redis-password
There will be no error

**Comment by mrniko** [comment]
can you share redisson logs for both configurations?

**Comment by mrniko** [comment, marked as answer]
Fixed in 3.24.2 version
